calendar.php

```diff
@@ -0,0 +1,83 @@
+<html>
+<head>
+    <link rel = 'stylesheet' href = 'style.css'/>
+</head>
+<body>
+
+<?php
+include ('include/initialize.php');
+
+$_SESSION['date'] = getDateToStoreInSession();
+$selectedMonthObject = $_SESSION['date'];
+
+$currentMonthObject = new DateTime(); //for checking if we are in current month for ln 54
+$currentMonthDisplayedAsNo = convertDateTimeToMonthNo($currentMonthObject);
```

Comment:
You could honestly just use the built-in PHP function ($currentMonthObject->format('n')) here rather than calling a function you created, "convertDateTimeToMonthNo," that essentially wraps the format function. You could apply this to other functions as well, but it's up to you -- it could really decrease the number of function declarations in `functions.php`. I understand if the function names are there for clarity since `format("n")` isn't really clear about what its purpose is

Reply:
I already wrote functions for other date formatting so I wanted to be consistent. But I do agree I just wrapped the function format. I will leave it as is for now because format('n') wasn't immediately clear to me.